plugins/color/gsd-color-state.c

```diff
@@ -20,6 +20,7 @@
 
 #include "config.h"
 
+#include <ctype.h>
```

Comment:
You made edits directly to the file instead of to the patch. If you can, use quilt and move this into the patch. If you cannot, let me know and I can take care of it later

Reply:
They should be moved to the patch file now.